abc.templates/base-workflows/contents/guardian-apply.yml

```diff
@@ -63,7 +63,7 @@ jobs:
         id: 'dirs'
         shell: 'bash'
         run: |-
-          DIRECTORIES=$(guardian entrypoints -detect-changes -source-ref=HEAD~1 -dest-ref=HEAD -format=json REPLACE_TERRAFORM_DIRECTORY)
+          DIRECTORIES=$(guardian entrypoints -dir=REPLACE_TERRAFORM_DIRECTORY -detect-changes -source-ref=HEAD~1 -dest-ref=HEAD -format=json)
```

Comment:
nit: can we wrap the dir in double quotes?

Reply:
done